src/parsers/directive.parser.ts

```diff
@@ -18,7 +18,10 @@ export class DirectiveParser extends AbstractTemplateParser implements ParserInt
 		template = this._normalizeTemplateAttributes(template);
 		$(template)
 			.find('[translate]')
-			.each((i: number, element: CheerioElement) => {
+            .contents() // get child nodes
+            .filter(function() { // only keep text nodes
+				return this.nodeType === 3; // node type 3 = text node
```

Comment:
You don't want html in your keys, you can have html in your translations, but not in your keys, there's no reason for it.
Also the idea here is that we only want to focus on first level children, anything beyond is another dom element and you can use translate on it if you want, but the directive won't touch it, it's too complicated otherwise.

But anyway with the existing code it doesn't take the html either becauseit uses `text()`, it will not use the html.